server/src/main/java/org/candlepin/dto/api/v1/ActivationKeyContentOverrideDTO.java

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright (c) 2009 - 2018 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.dto.api.v1;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import org.apache.commons.lang.builder.HashCodeBuilder;
+
+
+
+/**
+ * The ActivationKeyContentOverrideDTO is used as a DTO for the ActivationKeyContentOverride
+ * instances.
+ */
+public class ActivationKeyContentOverrideDTO extends ContentOverrideDTO<ActivationKeyContentOverrideDTO> {
+    public static final long serialVersionUID = 1L;
+
+    protected ActivationKeyDTO activationKey;
+
+
+    /**
+     * Initializes a new ActivationKeyContentOverrideDTO instance with null values.
+     */
+    public ActivationKeyContentOverrideDTO() {
+        // Intentionally left empty
+    }
+
+    /**
+     * Initializes a new ActivationKeyContentOverrideDTO instance which is a shallow copy of the
+     * provided source entity.
+     *
+     * @param source
+     *  The source entity to copy
+     */
+    public ActivationKeyContentOverrideDTO(ActivationKeyContentOverrideDTO source) {
+        this.populate(source);
```

Comment:
Should probably call `super(source);` instead of populate

Reply:
Fixed.